### PR TITLE
Adding CdnjsCatalog convert tests and npm package tests

### PR DESCRIPTION
**Pull request by YangLyu89**

### CODE REVIEW

File: test/LibraryManager.Test/Providers/Cdnjs/CdnjsCatalogTest.cs [hunk 1]
```diff
@@ -154,5 +154,79 @@ public void Setup()
...
+        }
+
+        [DataTestMethod]
+        [DataRow(null)]
+        [DataRow("")]
+        [DataRow(@"{""results"":[12}")]
+        public void ConvertToLibraryGroup_InvalidJsonCatalog(string json)
+        {
+         //   string json = @"{""results"":[12}";
+         //   string json = @"{""results"":[]}";
```
Comment (bhsubra) [inline test/LibraryManager.Test/Providers/Cdnjs/CdnjsCatalogTest.cs]: Remove?

**Comment by bhsubra** [comment]
>             var list = new List<Asset>();

Can we have a better name for this?maybe assets?

---
Refers to: src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs:278 in d787c74. [](commit_id = d787c74109d0398e0a40936539adc3d8b8a21f92, deletion_comment = False)

**Comment by YangLyu89** [comment]
>             var list = new List<Asset>();

Yes. That make sense to me.

---
In reply to: [407914986](https://github.com/aspnet/LibraryManager/pull/327#issuecomment-407914986) [](ancestors = 407914986)

---
Refers to: src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs:278 in d787c74. [](commit_id = d787c74109d0398e0a40936539adc3d8b8a21f92, deletion_comment = False)

**Comment by bhsubra** [comment]
>             }

As discussed offline, can you verify if this?

---
Refers to: src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs:299 in d787c74. [](commit_id = d787c74109d0398e0a40936539adc3d8b8a21f92, deletion_comment = False)

File: src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs [hunk 2]
```diff
@@ -311,6 +304,49 @@ private static string NormalizedGroupName(string groupName)
...
+
+                return null;
+            }
+        }
+
+        internal List<Asset> ConvertToAssets(string json)
+        {
+            try
+            {
+                List<Asset> list = new List<Asset>();
```
[the remark this answers is not in the document]
Comment (YangLyu89) [reply src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs]: Will do

---
In reply to: [205291743](https://github.com/aspnet/LibraryManager/pull/327#discussion_r205291743) [](ancestors = 205291743)

[the remark this answers is not in the document]
Comment (YangLyu89) [reply src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs]: assets is defined for IEnumerable<Asset>. Probably "assetList" or ""result" would be good.

---
In reply to: [205292471](https://github.com/aspnet/LibraryManager/pull/327#discussion_r205292471) [](ancestors = 205292471,205291743)

File: src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs [hunk 3]
```diff
@@ -309,7 +302,50 @@ private static string NormalizedGroupName(string groupName)
...
+
+                return null;
+            }
+        }
+
+        internal List<Asset> ConvertToAssets(string json)
+        {
+            try
+            {
+                List<Asset> assetList = new List<Asset>();
```
Comment (bhsubra) [inline src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs]: >assetList [](start = 28, length = 9)

nit: assets?

File: src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs [hunk 4]
```diff
@@ -309,7 +302,50 @@ private static string NormalizedGroupName(string groupName)
...
+            catch (Exception ex)
+            {
+                System.Diagnostics.Debug.Write(ex);
+
+                return null;
+            }
+        }
+
+        internal List<Asset> ConvertToAssets(string json)
+        {
```
[the remark this answers is not in the document]
Comment (YangLyu89) [reply src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs]: Applied. Thanks.

---
In reply to: [205314815](https://github.com/aspnet/LibraryManager/pull/327#discussion_r205314815) [](ancestors = 205314815)

**Pull request merged**